File: EstructurasControl/Condicionales/Multiples/Python/ejercicio2.py
print("=== Menú de Opciones ===")
print("1. Sumar")
print("2. Restar")
print("3. Multiplicar")
print("4. Dividir")

opcion = int(input("Elige una opción (1-4): "))

# Condicional múltiple
if opcion == 1:
    print("Elegiste: Sumar")
elif opcion == 2:
    print("Elegiste: Restar")
elif opcion == 3:
    print("Elegiste: Multiplicar")
elif opcion == 4:
    print("Elegiste: Dividir")
else:
    print("Opción inválida")

File: EstructurasControl/Condicionales/Multiples/Csharp/ejercicio2.cs
using System;

class ProgramaCondicionalMultiple2
{
    static void Main()
    {
        Console.WriteLine("=== Menú de Opciones ===");
        Console.WriteLine("1. Sumar");
        Console.WriteLine("2. Restar");
        Console.WriteLine("3. Multiplicar");
        Console.WriteLine("4. Dividir");
        Console.Write("Elige una opción (1-4): ");

        int opcion = Convert.ToInt32(Console.ReadLine());

        // Condicional múltiple
        if (opcion == 1)
        {
            Console.WriteLine("Elegiste: Sumar");
        }
        else if (opcion == 2)
        {
            Console.WriteLine("Elegiste: Restar");
        }
        else if (opcion == 3)
        {
            Console.WriteLine("Elegiste: Multiplicar");
        }
        else if (opcion == 4)
        {
            Console.WriteLine("Elegiste: Dividir");
        }
        else
        {
            Console.WriteLine("Opción inválida.");
        }
    }
}
